V15: Document Type Localization updates (#6909)
umbraco/UmbracoDocs: 15/umbraco-cms/fundamentals/data/defining-content/document-type-localization.md

* Create a new **tab** called `#tabs_content`.
* Add a new **group** called `#groups_titles`.
* Add a **property** called `#properties_title` with **alias** `title`.
  * Set description to `{#properties_title-desc}`.
  * Use a `TextString` editor.
  * Enable to `Set this field as mandatory`.
  * Under validation add `#properties_title-message`.

{% hint style="info" %}

> Please note, property descriptions support [Umbraco Flavored Markdown](../../../reference/umbraco-flavored-markdown). This requires a different syntax (wrapped in brackets), so not to conflict with Markdown header syntax.

Reviewer: ```suggestion
Property descriptions support [Umbraco Flavored Markdown](../../../reference/umbraco-flavored-markdown), which uses a different syntax (wrapped in brackets) to avoid conflicts with Markdown headers.
```

{% endhint %}

![Applying localization to a property](../images/localization-document-type-editor-validation.png)

* Add a **property** called `#properties_subTitle` with **alias** `subTitle`.
  * Set description to `{#properties_subTitle-desc}`.
  * Use a `TextString` editor.
* Enable to `Allow as root` in the **Permissions** tab.

![Applying localization to a Document Type](../images/localization-document-type-editor.png)

> 3. When creating and editing the content, you will see that the backoffice now uses the configured localizations.

Reviewer: Unfortunately, I do not see the localized content. This is what I see: 
![image](https://github.com/user-attachments/assets/fff6257b-4c29-427d-98d9-5f642a1055b5)
